io.catenax.fleet.vehicles/2.0.0/Vehicles.ttl

```diff
@@ -0,0 +1,395 @@
+   samm:preferredName "List of vehicles"@en ;
+   samm:description "List of vehicles"@en ;
+   samm:dataType :Vehicle .
+
+:MetaCharacteristic a samm:Characteristic ;
+   samm:preferredName "MetaCharacteristic"@en ;
+   samm:description "Chracteristic for the meta information."@en ;
+   samm:dataType :MetaInformation .
+
+:Vehicle a samm:Entity ;
+   samm:preferredName "Vehicle"@en ;
+   samm:description "Vehicle data that does not fit into one of the other entities."@en ;
+   samm:properties ( [ samm:property :recordStatus; samm:optional true ] :anonymizedVin [ samm:property :catenaXVehicleId; samm:optional true ] [ samm:property :class; samm:optional true ] [ samm:property :driveSystemPower; samm:optional true ] [ samm:property :driveType; samm:optional true ] [ samm:property :powerTrainType; samm:optional true ] [ samm:property :modelDescription; samm:optional true ] [ samm:property :modelIdentifier; samm:optional true ] [ samm:property :plantCountryCode; samm:optional true ] [ samm:property :plantDescription; samm:optional true ] [ samm:property :plantIdentifier; samm:optional true ] [ samm:property :productionDate; samm:optional true ] [ samm:property :softwareCategory; samm:optional true ] [ samm:property :softwareVersion; samm:optional true ] [ samm:property :soldCountryCode; samm:optional true ] [ samm:property :soldCountryGroup; samm:optional true ] [ samm:property :soldDate; samm:optional true ] [ samm:property :steeringPos; samm:optional true ] [ samm:property :vehicleSeries; samm:optional true ] [ samm:property :wmiCode; samm:optional true ] [ samm:property :wmiNameNHTSA; samm:optional true ] [ samm:property :engines; samm:optional true ] [ samm:property :equipments; samm:optional true ] ) .
+
+:MetaInformation a samm:Entity ;
+   samm:preferredName "Meta Information"@en ;
+   samm:description "This entity groups all meta information properties. Meta information properties give hints for a Catena-X data consumer to better understand, how this data set was created on the Catena-X data provider side."@en ;
+   samm:properties ( :selectionCriteria [ samm:property :selectionStart; samm:optional true ] [ samm:property :selectionEnd; samm:optional true ] ) .
+
+:recordStatus a samm:Property ;
+   samm:preferredName "Record status"@en ;
+   samm:description "The record operation enunumeration can be used to realize delta update concept.\nDelta updateupdate concept means:\n- You transfer an initial load of data\n- After the first week only the delta to the initial load is transferred\n\n\nThe record status describes whether this record is:\n- new=>This record is transferred the first time\n- update=> Some properties of this record have changed compared to a previous transfer\n- delete=> This record was transferred in the initial load or in a previous delta update, but is not \nused any more and therefore it should be deleted on data consumer side\n- same=> This record was transferred in the initial load or in a previous delta update"@en ;
```

Comment:
typo `enunumeration`, `updateupdate` , `\nused`.
please check in other models also if used same description.